src/common/models/essence/essence.ts

```diff
@@ -265,10 +264,9 @@ export class Essence extends ImmutableRecord<EssenceValue>(defaultEssence) {
 
   public getEffectiveFilter(
     timekeeper: Timekeeper,
-    { highlightId = null, combineWithPrevious = false, unfilterDimension = null }: EffectiveFilterOptions = {}): Filter {
-    const { dataCube, highlight, timezone } = this;
+    { combineWithPrevious = false, unfilterDimension = null }: EffectiveFilterOptions = {}): Filter {
```

Comment:
`unfilterDimension` should be handled outside this method - get effective filter, then just remove clause on call site